### PR TITLE
Update the Okta hosted guide to for an auth connector migration step.

### PR DESCRIPTION
**Pull request by mdwn**
The hosted guide now notes that the user should adjust the okta SSO connector to point to the `okta-requester` role if it already exists in the system.

### CODE REVIEW

File: docs/pages/application-access/okta/hosted-guide.mdx [hunk 1]
```diff
@@ -97,6 +97,23 @@ user groups and direct application assignments as Access Lists. This will ensure
 that permissions in Okta are modeled properly in Teleport, allowing Teleport users
 to see the same Okta applications that they would when viewing the Okta dashboard.
 
+<Admonition type="note">
+If you already have an SSO setup on Teleport that utilizes Okta, take this time to update the role mapping in the
```
Comment (r0mant) [inline docs/pages/application-access/okta/hosted-guide.mdx:101]: I'm not sure who this note is for? Is it for people who are setting up Okta integration for the first time (in which case it configure everything correctly automatically?), or for people with existing Okta integrations (in which case how are they going to see this and know they need to change something)?

Comment (mdwn) [reply docs/pages/application-access/okta/hosted-guide.mdx:101]: This note is for people with existing integrations. Let me reword this to make this clearer.

Comment (r0mant) [reply docs/pages/application-access/okta/hosted-guide.mdx:101]: How are they going to see this? If this is something they have to do, there should be an automated migration procedure IMO.

Comment (mdwn) [reply docs/pages/application-access/okta/hosted-guide.mdx:101]: Replaced with https://github.com/gravitational/teleport.e/pull/3647.
